Update to Nerd Fonts 3.0

vinnydiehl opened the pull request:
Most of the Material icons were broken with 3.0, and many have been added. Not only were the keycodes broken, many have been renamed or removed altogether.

This PR fixes all of these issues and adds over 5,000 icons. As far as I can find, this is the complete set. The name formatting is now consistent with the [official names of the icons](https://www.nerdfonts.com/cheat-sheet).

This list was taken from [this repository](https://github.com/rainstormstudio/nerd-icons.el/tree/main/data) and transposed with [this script](https://gist.github.com/vinnydiehl/8ba79ab80add22be534d3955e5ef000f).

vinnydiehl committed:
Update to Nerd Fonts 3.0

Most of the Material icons were broken, and many have been added.

doctorfree commented:
This repository seems to have fallen out of maintenance, the last commit about a year and a half ago. It's an excellent plugin and I want to continue to use it. My fork at https://github.com/doctorfree/cheatsheet.nvim includes several fixes and improvements including this PR.

When and if the original maintainer comes back online I will submit PRs to sync but until then please feel free to use and/or contribute to my fork.

vinnydiehl commented:
@doctorfree Thank you! I didn't know about your fork, I've switched my config to use yours and will leave this issue open so that anyone searching for this compatibility fix may find it. I have also updated the [Telescope wiki](https://github.com/nvim-telescope/telescope.nvim/wiki/Extensions) to point to your fork, of course with a reference to this repository as the original plugin.

duraki commented:
@doctorfree: thanks a lot for the fork and your updates! Gotta check this out. You should enable **Issues** in your repository fork (@doctorfree).

@vinnydiehl: thanks a lot for the PR. I'll fetch the yours, upstream repo.

doctorfree commented:
@duraki thanks for the heads up. I enabled issues in my fork.

duraki commented:
Awesome man. Thankz, that was fast. I won't spam on this PR anymore, just wanted to note I will credit your fork as much. Bless u 🙏
